modules/consensus/accept_test.go

```diff
@@ -650,17 +639,15 @@ func (cst *consensusSetTester) testFileContractsBlocks() error {
 		}},
 	}
 	copy(txn.StorageProofs[0].Segment[:], segment)
-	err = cst.tpool.AcceptTransaction(txn)
+	err = cst.tpool.AcceptTransactionSet([]types.Transaction{txn})
```

Comment:
I think it would be pretty rare to see anything other than `Accept(tx)` or `Accept(txns...)`. But perhaps it's not worth the loss in clarity.